frontend/src/api/PublicAPI.js

```diff
@@ -24,3 +24,9 @@ export const sendForgotPasswordEmail = (email) =>
 
 export const getCurrentUser = (token) =>
   Api.get('/users/current', { headers: { Authorization: `Bearer ${token}` } });
+
+export const resetPassword = (token, newPassword) =>
```

Comment:
I might be too tired to follow logic here, but don't we need to pass `oldPassword` as well?

Reply:
Reset password only requires the user's Reset JWT token and their new password, because it is used after the "Forgot password?"

Maybe you're confusing this one with Aurimas' updateCredentials which he did yesterday. There we require the user to enter the old password

Reply:
Wait so we aren't using the same endpoint for update credentials and reset password?

Reply:
No